manage_people/static/manage_people/css/manage_people.css

```diff
@@ -281,12 +280,12 @@ form .fa-asterisk {
 label {
   color: #333333;
   font-size: 1em;
-  font-family: "HelveticaNeue-Bold", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: bold;
```

Comment:
As discussed offline, was hoping this would imitate the preferred `font-family`. I didn't check how many of these style rules were actually used in our app, but a quick eyeballs-on verified it looked ok! Thanks.